cylc/uiserver/graphql/tornado_ws.py

```diff
@@ -0,0 +1,381 @@
+# This Cylc solution for graphql subscriptions over websockets is essentially
+# a condense from of the stagnant graphql-ws project:
+# https://github.com/graphql-python/graphql-ws
+# and our implementation of the tornado integration PR that never got merged:
+# https://github.com/graphql-python/graphql-ws/pull/25/files
```

Comment:
TODO: Clear licencing implications.

Reply:
The modifications are reasonably quite large, but yes, still need to clear..